torch/jit/_script.py

```diff
@@ -53,6 +53,57 @@
     def Attribute(value, type):  # type: ignore
         return value
 
+Attribute.__doc__ = """
+    This method is a pass-through function that returns `value`. It is mostly
+    used to indicate to the TorchScript compiler that the left-hand side
+    expression is a class attribute with type of `type`.
```

Comment:
class attribute -> class and instance attribute

Reply:
Thanks for catching, I actually wanted to say "instance attribute". Done.